README.md

```diff
@@ -32,6 +32,17 @@ The following keybindings are available:
 - `RET` or `v`: Go to the entry at point
 - `q`: Quit pass
 
+#### 2FA / OTP Support
+
+If you have the [`pass-otp`](https://github.com/tadfisher/pass-otp) extension
+installed, you will be able to use the following keybindings as well:
+
+- `o i`: Insert an OTP key URI in a new entry
+- `o a`: Append an OTP key URI to an existing entry
+- `o s`: Take a screenshot of a OTP QR Code and have its related URI be appended to an existing entry
+- `o t`: Copy OTP token for entry at point
```

Comment:
this looks like the shortcut that people  will do again and again. What about `o o` then? This is easier to remember and type I think.

Reply:
Great idea!